pkg/check/privileges.go

```diff
@@ -0,0 +1,75 @@
+package check
+
+import (
+	"github.com/rvanderp3/vsphere-priv-check/pkg/util"
+	"k8s.io/klog/v2"
+	"time"
+)
+
+var (
+	lastError          error     = nil
+	showNextErrorAfter time.Time = time.Now()
+)
+
+const (
+	installConfigName = "install-config"
+)
+
+func shouldShowError() bool {
+	return time.Now().After(showNextErrorAfter)
+}
+
+func showError(err error) {
+	backoffDuration, _ := time.ParseDuration("4h")
+	showNextErrorAfter = time.Now().Add(backoffDuration)
+	klog.V(2).Infof("CheckUserPrivileges: %s", err)
+}
+
+// CheckUserPrivileges informational check which logs missing user privileges
+func CheckUserPrivileges(ctx *CheckContext) error {
```

Comment:
Would it be possible to add an unit test for this using a simulated vCenter? Check the other checks and xml files in `pkg/check/testdata`. Not sure if it simulates all the `FetchUserPrivilegeOnEntities` calls. But it would be cool, as we cannot test it too much in our real vCenter.

Reply:
I couldn't find where the simulator supported this type of simulation.  Unit tests are definitely something we'll add here though.

Reply:
Yes, we can add a mock for FetchUserPrivilegeOnEntities